torch_xla/runtime.py

```diff
@@ -313,8 +324,8 @@ def initialize_cache(path: str, readonly: bool = False):
   before any computations have been performed.
 
   Args:
-    path: The path at which to store the persistent cache.
-    readonly: Whether or not this worker should have write access to the cache.
+    path (str): The path at which to store the persistent cache.
```

Comment:
I don't think it's necessary to include the type in the docstring when it's already annotated. (`readonly` is an optional bool, not just a bool)

Reply:
yea then the type annotation is also incorrect haha, let me fix that in a follow up pr.